### PR TITLE
Fix access token for genome visualizations in public studies

### PR DESCRIPTION
**Pull request by eweitz**
This enables any signed-in users to see genome visualizations in public studies.  Previously for such public visualizations, signed-in users would need "view" permission explicitly granted to them by the study owner.  That is too restrictive.

An update to a functional test verifies this fix.  I've locally run functional UI tests related to this feature (`ruby test/ui_test_suite.rb -n /front-end.*(ideogram|igv).*/ ...`) and they pass.

This also fixes an edge-case bug in our UI test suite in which a race condition sometimes caused the wrong option to be selected.

### CODE REVIEW

**Comment by eweitz** [comment]
I'm fixing the [detected code style issues](https://app.codacy.com/app/Portals/single_cell_portal_core/pullRequest?prid=3414697) now.  It seems like a tab-vs-space issue in my editor.

File: test/ui_test_suite.rb [hunk 1]
```diff
@@ -4947,8 +4958,8 @@ def teardown
           file_type.send_keys('Metadata')
         when bam_file
           file_type.send_keys('BAM')
-          species_dropdown = form.find_element(:id, 'study_file_taxon_id')
-          species_dropdown.send_keys('mouse') # from lib/assets/default_species_assemblies.txt
+          # TODO: Determine where else this is needed, apply there.
+          select_option_from_dropdown(form, 'study_file_taxon_id', 'mouse')
```
Comment (bistline) [inline test/ui_test_suite.rb]: Really like this.  We should create a ticket in the backlog to apply this to all select boxes in the UI test suite.

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -300,26 +300,25 @@ def email_as_id(email)
...
-	# 	else
-	# 		if user.present?
-	# 			user.valid_access_token[:access_token]
-	# 		else
-	# 			nil
-	# 		end
-	# 	end
-	# end
-
-	# return an access token for viewing GCS objects client side, for signed-in users
```
Comment (knapii-developments) [inline app/helpers/application_helper.rb]: Why is this all commended out? Why not just remove it all?

File: test/ui_test_suite.rb (hunk 1 above)
Comment (eweitz) [reply test/ui_test_suite.rb]: Done! https://broadinstitute.atlassian.net/browse/SCP-1572

**Comment by eweitz** [comment]
Thanks, Jon and Eno.  I'll draft release notes for this.
